### ISSUE TITLE
ModuleNotFoundError: No module named 'xml.etree.cElementTree'

### ISSUE THREAD

**Issue by tim77** [flightlessmango/MangoHud#213]
This issue appeared in new `0.4.1` only in Fedora Rawhide (somewhat development forge and bleeding edge). On F32 and F31 all fine. I guess this could because on new Python 3.9 but don't know yet and we need to investigate this.

- [Full build log](https://kojipkgs.fedoraproject.org//work/tasks/6519/45596519/build.log)
- [Koji task](https://koji.fedoraproject.org/koji/taskinfo?taskID=45596041)

```meson
[1/42] /usr/bin/python3 ../bin/gen_enum_to_str.py --xml /usr/share/vulkan/registry/vk.xml --outdir /builddir/build/BUILD/MangoHud-0.4.1/x86_64-redhat-linux-gnu
FAILED: vk_enum_to_str.c vk_enum_to_str.h 
/usr/bin/python3 ../bin/gen_enum_to_str.py --xml /usr/share/vulkan/registry/vk.xml --outdir /builddir/build/BUILD/MangoHud-0.4.1/x86_64-redhat-linux-gnu
Traceback (most recent call last):
  File "/builddir/build/BUILD/MangoHud-0.4.1/x86_64-redhat-linux-gnu/../bin/gen_enum_to_str.py", line 28, in <module>
    import xml.etree.cElementTree as et
ModuleNotFoundError: No module named 'xml.etree.cElementTree'
```

**Comment by jackun** [comment]
> # Deprecated alias for xml.etree.ElementTree

Probably.

**Comment by jackun** [comment]
Fix in db070816174b4b84d0859ac5e29ef71520376d01

**Comment by tim77** [comment]
https://github.com/flightlessmango/MangoHud/commit/db070816174b4b84d0859ac5e29ef71520376d01 fixes this issue. Thanks!
